Bergado_Shamma/script/countries.js

```diff
@@ -0,0 +1,97 @@
+function searchCountry() {
+  let countryName = document.getElementById("country_input").value.trim();
```

Comment:
Try changing all the let to const. These codes are assigned once and are never reassigned. Using const ensures that the values are supposed to remain constant.